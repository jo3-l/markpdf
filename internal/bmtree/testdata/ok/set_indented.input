p1. a
    set 3
    p2. b
    set 1
    p1. c
p2. e