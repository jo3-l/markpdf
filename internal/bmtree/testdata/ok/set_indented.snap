- "a" @ p1
	- "b" @ p4
	- "c" @ p1
- "e" @ p2